spec/models/profile_model_spec.rb

```diff
@@ -89,4 +89,66 @@
       end
     end
   end
+
+  describe "ClassMethods" do
+    before :all do
+      DatabaseCleaner.start
```

Comment:
it is available for all specs. But you do have to set it up in before :all/after :all individually.